group-generators/helpers/data-providers/farcaster/index.ts

```diff
@@ -0,0 +1,95 @@
+import axios from "axios";
+import { retryRequest } from "@group-generators/helpers/data-providers/utils/utils";
+import { FetchedData } from "topics/group";
+
+export class FarcasterProvider {
+  url: string;
+  headers: {
+    authorization: string;
+    accept: string;
+  };
+
+  public constructor() {
+    this.url = "https://api.farcaster.xyz/v2/";
+    this.headers = {
+      authorization: process.env.FARCASTER_API_KEY as string,
+      accept: "application/json",
+    };
+  }
+
+  public async getLastCreatedFid(): Promise<number> {
+    const { data: res } = await axios({
+      url: this.url + "recent-users",
+      method: "get",
+      headers: this.headers,
+    });
+
+    if (Object(res).result) {
```

Comment:
Why not create a type `FarcasterUser` so that you can handle your types without using `Object(res)` ? I think it will also add readability for people trying to understand the type returned by the farcaster api for this request

Reply:
Yes totally!